packages/core/core/src/RequestTracker.js

```diff
@@ -1157,12 +1161,14 @@ export default class RequestTracker {
       return result;
     } else if (node.resultCacheKey != null && ifMatch == null) {
       let key = node.resultCacheKey;
-      invariant(this.options.cache.hasLargeBlob(key));
-      let cachedResult: T = deserialize(
-        await this.options.cache.getLargeBlob(key),
-      );
-      node.result = cachedResult;
-      return cachedResult;
+      try {
```

Comment:
because we're clearing entries on write, reading an asset graph while a new version for the same key is getting written will trigger an error here

Reply:
If we don't clean-up then we don't need to handle this, but the state might still be no good as you read a potentially stale file out of disk.